### PR TITLE
Add planning call notes and update milestones

### PR DESCRIPTION
**Pull request by dcwalk**
Okay, per our meeting, updated milestones here and in calendar... added to readme.

Can we async review/leave comments and okay here?

### CODE REVIEW

**Comment by dcwalk** [comment]
bump @garrying and @ana0 ! would be good to have signed off before Thursday :bowing_woman:

File: notes/2019-03-30-planning-call.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+- garry
+- sarah
+- dawn
+- ben
+
+# Agenda
+
+- Old Business
+    - [x] Dawn to follow up this week on venue
+    - [x] Ben to sort livestreaming with patcon for March 28 meetup <-- dc: we discussed trackin on gh issue once I followed up
```
Comment (garrying) [inline notes/2019-03-30-planning-call.md]:
```suggestion
    - [x] Ben to sort livestreaming with patcon for March 28 meetup <-- dc: we discussed tracking on GH issue once I followed up
```

File: notes/2019-03-30-planning-call.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+- 2 hours a short amount of time
+- dealing with livestream?
+    - flow: microphone enabled
+    - screen is blown out (quality with camera?)
+    - split by talk 
+    - using wifi choppy, post will be local capture so avoid some of that
+
+## Venue Updates
+
+- Email: Gammaspace working on Friday
```
Comment (garrying) [inline notes/2019-03-30-planning-call.md]:
```suggestion
- Email: Gamma Space working on Friday 
```

File: notes/2019-03-30-planning-call.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    - [x] Dawn move over the p2p issue in 2018 repo
+    - [x] Sarah to set up next grant doc
+    - [x] Sarah to check on grant results deadlines
+    - [x] Sarah review sponsorship package
+- Standing Business
+  - Run through ["In Progress"](https://github.com/ournetworks/2019/projects/1)
+  - Run through ["Calendar Deadlines"](https://calendar.google.com/calendar/embed?src=aers7atolh0uurlfmkoki9kikg%40group.calendar.google.com&ctz=America%2FToronto)
+- Other Business
+  - Venue Update
+  - Milestone//dates
```
Comment (garrying) [inline notes/2019-03-30-planning-call.md]:
```suggestion
  - Milestones/dates
```

File: notes/2019-03-30-planning-call.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+        04/01 conference open hour #1 (-25)
+        05/01 2019 website relaunch, marketing materials (-21)
+        05/15 conference open hour #2 (-19)
+        06/01 keynote speaker(s) (-17)
+        06/01 CFP closes (-17)
+        06/15 registration launched (-15)
+        07/01 submission decisions (-12) 
+        07/25 schedule posted (-9)
+- Shift CFP closing until July 1?
+- Fist week July block off for submission dec'ns
```
Comment (garrying) [inline notes/2019-03-30-planning-call.md]:
```suggestion
- First week July block off for submission dec'ns
```

File: notes/2019-03-30-planning-call.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+- Smaller/more intentful outreach for exhibition mid-June
+    - If we get one of the Arts Council grants, do outreach end of June to invite specific artists to CFP
+
+## Sponsorship 
+
+- We sold 100 last year, aim for 150 attendees this year
+- Drop RSVP chair from perk
+- ben: Hand-drawn poster or marketing retractable!
+- Outreach n April for previous sponsers
```
Comment (garrying) [inline notes/2019-03-30-planning-call.md]:
```suggestion
- Outreach in April for previous sponsors
```

File: notes/2019-03-30-planning-call.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    - buying thrift shirts is cheaper/similar to new shirts at costs 
+- "our networks like our cities built on top of ruins"
+- photocopier photocopy for zine library (filesharing reference)
+- repair?
+    - forcing a confrontation between supply chains and infrastructure
+- collect e-waste recycling
+    - donated networking things 
+    - artistic work?
+    - related to Ryan's thing at FreeGeek? cc. Garry?
+- use a hopes and dreams gh issue
```
Comment (garrying) [inline notes/2019-03-30-planning-call.md]:
```suggestion
- use a hopes and dreams GH issue
  - https://github.com/ournetworks/2019/issues/34
```

**Comment by dcwalk** [comment]
Thanks @garrying, merged them all!

**Comment by dcwalk** [comment]
Merging so it can be referenced more easily!

**Pull request merged**